omero/users/cli/containers-annotations.txt

```diff
@@ -0,0 +1,63 @@
+Creating containers and annotations
+-----------------------------------
+
+The :omerocmd:`obj` command allows users to create and update OMERO objects,
+this can be used to create containers, i.e. projects, datasets and screens. It
```

Comment:
Since this is very much limited to pre-import (containers) and post-import (annotations) should folders be mentioned?